datasets/_global/eiti_soe_list/crawler.py

```diff
@@ -0,0 +1,39 @@
+from typing import Any, Dict
+from zavod import Context
+
+
+def crawl_entity(context: Context, item: Dict[str, Any]) -> None:
+    entity = context.make("LegalEntity")
+
+    soe = item.pop("SOE")
+    year = item.pop("Year")
+    country = item.pop("Country")
+    fin_statement = item.pop("Audited Financial Statement or Equivalent")
+    website = item.pop("Public Listing or Website")
+
+    # Making a unique ID for the entity
+    entity.id = context.make_id(soe, year, country)
```

Comment:
I think let's drop year from key so that the same SOE over multiple years gets the same key. Or did you include it because there's some risk of different SOEs having the same name in the same country over time?